test/unit/processing/parse-payment-file.test.js

```diff
@@ -4,24 +4,25 @@ const { sendBatchProcessedEvents, sendBatchErrorEvent } = require('../../../app/
 jest.mock('../../../app/messaging')
 const { sendPaymentBatchMessage } = require('../../../app/messaging')
 
-jest.mock('../../../app/parse-siti-payment-file/build-payment-file')
-const { buildAndTransformParseFile } = require('../../../app/parse-siti-payment-file/build-payment-file')
+jest.mock('../../../app/processing/get-payment-requests-from-file')
+const getPaymentRequestsFromFile = require('../../../app/processing/get-payment-requests-from-file')
 
-const { parsePaymentFile } = require('../../../app/parse-siti-payment-file/parse-payment-file')
+const parsePaymentFile = require('../../../app/processing/parse-payment-file')
+const { sfiPilot } = require('../../../app/schemes')
 
+let sequence
 let filename
 let fileBuffer
-let sequence
 
 let mockPaymentRequests
```

Comment:
this isn't defined anywhere yet it is used as an expected input to functions and doesn't fail?! I suspect this is what the `getPaymentRequestsFromFile.mockResolvedValue` should be?

Reply:
It's set in the `beforeEach` to `1` by default.  A couple of the later scenarios overwrite it.